frontend/public/components/storage/create-pvc.tsx

```diff
@@ -200,35 +166,16 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
           {t('public~A unique name for the storage claim within the project')}
         </p>
       </div>
-      <label className="control-label co-required" htmlFor="access-mode">
-        {t('public~Access mode')}
-      </label>
       <div className="form-group">
-        {getAccessModeRadios().map((radio) => {
-          let radioObj = null;
-          const disabled = !allowedAccessModes.includes(radio.value);
-
-          allowedAccessModes.forEach((mode) => {
-            const checked = !disabled ? radio.value === accessMode : radio.value === mode;
-            radioObj = (
-              <RadioInput
-                {...radio}
-                key={radio.value}
-                onChange={handleAccessMode}
-                inline={true}
-                disabled={disabled}
-                checked={checked}
-                aria-describedby="access-mode-help"
-                name="accessMode"
-              />
-            );
-          });
-
-          return radioObj;
-        })}
-        <p className="help-block" id="access-mode-help">
-          {accessModeHelp}
-        </p>
+        <AccessModeSelector
+          className=""
```

Comment:
You should be able to remove this prop instead of setting it to `""`

Reply:
Ah! i missed this twice, let me fix this correctly this time